templates/cassiopeia/scss/blocks/_footer.scss

```diff
@@ -12,10 +14,10 @@
   .back-top {
     align-self: flex-end;
     height: 40px;
-    padding: 8px 11px;
-    color: $cassiopeia-template-color;
-    background: $white;
-    border-radius: 3px;
+    padding: $cassiopeia-grid-gutter/2;
+    color: var(--cassiopeia-color-primary);
+    background: color("white");
```

Comment:
is there a difference between color("white") and $white

Reply:
That is related with the functions from bootstrap:
https://getbootstrap.com/docs/4.0/getting-started/theming/#functions
I'm not sure which syntax is better...

Reply:
If they work the same then it would be best if we standardise on one

Reply:
@brianteeman Agree. We have to fix that. I've created issue #73 so we don't forget about that. For this PR it should not be a show stopper, so I mark the conversation as resolved. Thank you very much for your reviews, it is really appreciated.

Reply:
fixed in #73